ui/app/templates/components/masked-input.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}}" data-test-masked-input>
+	{{#if displayOnly}}
+		<pre class="masked-value display-only">{{displayValue}}</pre>
+	{{else}}
+  <textarea
+    class="input masked-value"
+    rows=1
+    wrap="off"
+    placeholder="value"
+    onfocus={{action "setFocus" true}}
+    onblur={{action "setFocus" false}}
```

Comment:
I think these could be `{{action (mut setFocus) true}}` and `…false}}`, removing the need for the action.